### PR TITLE
[WIP] nukie rats

### PR DESCRIPTION
**Pull request by ynot01**
#### Changelog

:cl:  
rscadd: Added a chance (1/7) for the mice migration event to instead summon Mouse Operatives
/:cl:

### CODE REVIEW

**Comment by Xoxeyos** [comment]
e x p l a i n

**Comment by ynot01** [comment]
they have 20hp, don't get shocked from eating wires, and gib on death

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 1]
```diff
@@ -90,6 +89,24 @@
...
+	minbodytemp = 0
+	maxbodytemp = 5000
+	maxHealth = 20
+	health = 20
+	mob_biotypes = list(MOB_INORGANIC)
+
+/mob/living/simple_animal/mouse/mouse_op/New()
+	..()
+	desc = "Oh no..."
+	icon_state = "mouse_operative"
```
Comment (Lostlarry) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm:101]: why isn't this preset value in class defintion?

Comment (ynot01) [reply code/modules/mob/living/simple_animal/friendly/mouse.dm:101]: vg shitcode

**Comment by oklews** [comment]
speedmerge please

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 2]
```diff
@@ -90,6 +89,22 @@
...
+	health = 20
+	mob_biotypes = list(MOB_INORGANIC)
+	desc = "Oh no..."
+	icon = 'yogstation/icons/mob/animal.dmi'
+	icon_state = "mouse_operative"
+
+/mob/living/simple_animal/mouse/mouse_op/death(var/gibbed = FALSE, toast)
+	. = ..()
+	if(gibbed == FALSE)
+		src.gib()
```
Comment (Lostlarry) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm:106]: you have problem with the original death proc that will produce a mouse corpse and qdel itself before you try to gib it

**Comment by Nickvr628** [comment]
Speedmerge

**Comment by fluffe9911** [comment]
neat

**Comment by Firewolf34** [comment]
Get this in the codebase immediately

**Comment by alexkar598** [comment]
S. T. A. L. E.

**Pull request closed without merging**